lib/tasks/factory_girl.rake

```diff
@@ -0,0 +1,15 @@
+namespace :factory_girl do
```

Comment:
@Kevin-Wei copy that. So you're recommending it just be a CI task rather than a user task it sounds like. When we merge I'll add an issue so that we remember to add it to Circle's tasks.

Adding Guard sounds like a good candidate for backlog once we've hit MVP. Agree that it's too heavy for where we are at now.